### PR TITLE
Code documentation cleanup

### PR DESCRIPTION
**Pull request by nllong**
#### Any background context you want to provide?
Just making the docstrings consistent.

### CODE REVIEW

File: management/uo_des.py [hunk 1]
```diff
@@ -62,8 +62,8 @@ def build_sys_param(
     overwrite: bool,
     microgrid: bool,
 ):
-    """
-    Create system parameters file using uo_sdk output
+    """Create system parameters file using uo_sdk output
+    #FIXME: Formatting of the args...
```
Comment (vtnate) [inline management/uo_des.py:66]: What needs to be formatted in the cli args?

**Review by vtnate — APPROVED** [review]
Standardization ftw! Do we want to go farther in improving docstrings at this time? If we wanna get crazy and do code examples in docstrings, at least [ruff knows how to handle that](https://docs.astral.sh/ruff/formatter/#docstring-formatting)

File: geojson_modelica_translator/geojson/urbanopt_geojson.py [hunk 2]
```diff
@@ -17,8 +17,7 @@ class GeoJsonValidationError(Exception):
 
 # TODO: Inherit from GeoJSON Feature class, move to its own file
 class UrbanOptLoad:
-    """
-    An UrbanOptLoad is a container for holding Building-related data in a dictionary. This object
+    """An UrbanOptLoad is a container for holding Building-related data in a dictionary. This object
```
[the remark this answers is not in the document]
Comment (nllong) [reply geojson_modelica_translator/geojson/urbanopt_geojson.py:20]: Yes, you are 100% correct. But I wasn't going to rewrite the docs, just better format.

File: management/uo_des.py (hunk 1 above)
Comment (nllong) [reply management/uo_des.py:66]: The args in the doctoring need to be formatted differently... 

hit cmd-shift-2 after the function and it should stub it out for you (in VScode)

**Pull request merged**